src/librustc_driver/driver.rs

```diff
@@ -1131,7 +1135,7 @@ where
     });
 
     time(sess, "early lint checks", || {
-        lint::check_ast_crate(sess, &krate, false)
+        lint::check_ast_crate(sess, &krate, false, rustc_lint::BuiltinCombinedEarlyLintPass::new())
```

Comment:
This is not the same instance used to register the lints. I don't think that matters for the lints in rustc though.